docs/Community/sig-release/charter.md

```diff
@@ -0,0 +1,47 @@
+SIG Release Charter
+===================
+
+## Scope
+
+SIG Release is responsible for the generation of periodic numerical binary releases of the Open Enclave SDK project. This includes the processes necessary for tracking feature delivery and coordinating various teams' execution towards planned milestones.
+
+## In Scope
+
+### Code, Binaries, or Services
+
+- Creation of numerical release tags in GitHub, and their application to issues and PRs
+- Maintenance of kanban-style boards for the N+1 release tracking
+- Creation of release notes, when a release is published, and inclusion of those notes in the appropriate locations (e.g., in Git and on the website)
+- Validation of the output of the build system (e.g., final qualification of the binary outputs of a CI/CD pipeline)
+
+### Cross-cutting or Externally Facing Processes
+
+- Establishing meetings, as needed, to unblock any at-risk work items in the lead up to a release
+- Monitoring the build system and coordinating reviews leading up to cutting a release
+- Ensuring that releases are cut on time, or alerting the CGC if there will be delays in the release process
+
+### Out of Scope
+
+SIG-Release is *not* responsible for actually writing code, fixing tests, or unbreaking CI. Those deliverables are the responsibility of their respective SIGs
+
+## Process & Management
+
+SIG-Release meets only as needed. Meetings are generally more frequent in the run-up to a release.
+
+Meetings are held at PLACE, with AGENDA/MINUTES available or not. Meetings are OPEN/CLOSED.
```

Comment:
@AevaOnline Sig-release may not need to meet. Most of the work is on Github. It does need the CI/CD team involvement to generate and validate packages. To that end, I would like @BRMcLaren to be included in this SIG as well.

Reply:
@radhikaj is there a check-in or sign-off process for cutting a release? An informal discussion that happens between you and @BRMcLaren when generating and validating the packages? If so, I'd say that's a "meeting". 

It doesn't need to be a regularly scheduled meeting, but in the interest of building community engagement, I think it should be documented, communicated, and open to the community.

An example of this might be:

> There is usually just one release meeting held between the sig-release chair and the sig-cicd chair during a given release cycle, and it is usually held the week before the release. As 99% of the work happens on github, this meeting is only a brief touch point to validate expectations and confirm that the builds are ready to be cut and finalized. The meeting will be announced at least three days ahead of time on the mailing list.

If there is truly no need for any discussion about the release readiness, and all coordination happens on github, that's also perfectly in keeping with the Principles,  and I would just ask for it to be documented in this charter so that the process is transparent. Something like:

> SIG-Release does not hold any meetings. All preparation for releases happens via github issues, tags, and tracking boards. (insert link to document of the process)

Reply:
The process for release is - SIG-testing generates RC packages. We do an RC release and send out an email to the DL requesting feedback. The OE SDK core team tests the OE SDK packages manually. SIG-Testing generates final release packages once the release manager says the release branch is in its final state. We do another round of manual testing. We then make a final release.